web/src/pages/SSL/List.tsx

```diff
@@ -109,8 +107,8 @@ const Page: React.FC = () => {
         actionRef={tableRef}
         request={fetchList}
         search={{
-          searchText: formatMessage({ id: 'component.global.search' }),
-          resetText: formatMessage({ id: 'component.global.reset' }),
+          searchText: `${formatMessage({ id: 'component.global.search' })}`,
+          resetText: `${formatMessage({ id: 'component.global.reset' })}`,
```

Comment:
why not use formatMessage({ id: 'component.global.reset' }) directly?

Reply:
yes. you are right. 
Infact when I debuged this feature, use `formatMessage({ id: 'component.global.reset' })` directly could not worked well, so I used thie way instead, and I could not figured out why. But now it worked, any way, fixed